tests/MongoDB.Analyzer.Tests.Common.TestCases/Linq/LinqBsonAttributes.cs

```diff
@@ -22,15 +22,15 @@ namespace MongoDB.Analyzer.Tests.Common.TestCases.Linq
 {
     public sealed class LinqBsonAttributes : TestCasesBase
     {
-        [MQL("aggregate([{ \"$match\" : { \"Cost\" : 22.5 } }])")]
-        [MQL("aggregate([{ \"$project\" : { \"Name\" : \"$_id\", \"_id\" : 0 } }])")]
-        [MQL("aggregate([{ \"$sort\" : { \"Weight\" : 1 } }])")]
-        [MQL("aggregate([{ \"$match\" : { \"Color\" : \"Red\" } }])")]
-        [MQL("aggregate([{ \"$group\" : { \"_id\" : \"$ExpiryDate\" } }])")]
-        [MQL("aggregate([{ \"$match\" : { \"Price\" : { \"$lt\" : 20.0 } } }, { \"$skip\" : 2 }])")]
-        [MQL("aggregate([{ \"$match\" : { \"GreenAppleCost\" : { \"$lt\" : 35.0 } } }, { \"$project\" : { \"Name\" : \"$_id\", \"TotalCost\" : \"$Cost\", \"_id\" : 0 } }, { \"$skip\" : 5 }])")]
-        [MQL("aggregate([{ \"$match\" : { \"AppleType\" : \"Green Apple\" } }, { \"$project\" : { \"Name\" : \"$_id\", \"TotalCost\" : \"$Cost\", \"_id\" : 0 } }, { \"$limit\" : 5 }])")]
-        [MQL("aggregate([{ \"$match\" : { \"Quantity\" : DateTimeOffset.Now.Day } }, { \"$project\" : { \"TimeSpanField\" : \"$TimeSpanField\", \"DateTimeOffset\" : \"$DateTimeOffset\", \"_id\" : DateTimeOffset.Now.Day } }])")]
+        [MQL("Aggregate([{ \"$match\" : { \"Cost\" : 22.5 } }])")]
+        [MQL("Aggregate([{ \"$project\" : { \"Name\" : \"$_id\", \"_id\" : 0 } }])")]
+        [MQL("Aggregate([{ \"$sort\" : { \"Weight\" : 1 } }])")]
+        [MQL("Aggregate([{ \"$match\" : { \"Color\" : \"Red\" } }])")]
+        [MQL("Aggregate([{ \"$group\" : { \"_id\" : \"$ExpiryDate\", \"_elements\" : { \"$push\" : \"$$ROOT\" } } }])")]
```

Comment:
What it this $push $$ROOT all about?

Reply:
This is also part of LINQ3 machinery for `GroupBy` translation.
`_elements` are later deserialized as `System.Linq.IGrouping<,>`.